### ISSUE TITLE
Can I control source media stream?

### ISSUE THREAD

**Issue by progre** [jeeliz/jeelizWeboji#3]
Hi. It looks like very useful app.

I'll think when I use it, I want to control inner of `jeelizFaceTransfer.js`. for example, controling timing of `getUserMedia()`, using remote media stream, or using recorded video of my face.
But I can't judge possibility of them because it's source code isn't public.
How should I control it? or another solution, could I see sourcecode of jeelizFaceTransfer.js?

thanks

**Comment by xavierjs** [comment]
Hi,

I have improved the library on the last commit.
I have added the possibility to use an external `<video>` element rather than the current camera one. You There is the link to the live demo and source code from the README of the github (search for `Cute fox from a MP4 video`)
It is now possible to specify a specific device ID or to change the input resolution.

The WebRTC helper is the same than this one: https://github.com/jeeliz/jeelizAR/blob/master/helpers/JeelizMediaStreamAPIHelper.js
except it is included and compacted in the main script.
Of course if you use your own `<video>` element you don't need the helper.
